modules/programs/havoc.nix

```
@@ -0,0 +1,69 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.havoc;
+  iniFormat = pkgs.formats.ini { };
+
+in {
+  meta.maintainers = with lib.maintainers; [ AndersonTorres ];
+
+  options.programs.havoc = {
+    enable = mkEnableOption "Havoc terminal";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.havoc;
+      defaultText = literalExpression "pkgs.havoc";
+      description = "The havoc package to install";
+    };
```

There is now a simpler and more concise way of declaring a package option:
```suggestion
    package = mkPackageOption pkgs "havoc" {};
```

---

Done!

---

@berbiche ping